IntegrationTests/XCTestPlans/AWSIntegrationTestsOnCI.xctestplan

```diff
@@ -17,43 +17,43 @@
     {
       "target" : {
         "containerPath" : "container:",
```

Comment:
A lot of churn in this file, but the only material change is that the DynamoDB test bundle is added to the test plan.